homeassistant/components/bosch_shc/sensor.py

```diff
@@ -103,7 +103,9 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
             )
         )
 
-    for sensor in session.device_helper.smart_plugs:
+    for sensor in (
+        session.device_helper.smart_plugs + session.device_helper.light_switches
```

Comment:
Is this change required for the version bump? Otherwise please move it to a separate PR.

Reply:
I moved this second fix to another PR